Simulator.lua

```diff
@@ -1526,6 +1526,23 @@ local function createSimulator(a_Options, a_Logger)
 				type = io.type,
 				write = io.write,
 			},
+			debug = 
+			{
+				getupvalue = debug.getupvalue,
+				debug = debug.debug,
+				sethook = debug.sethook,
+				getmetatable = debug.getmetatable,
+				gethook = debug.gethook,
+				setmetatable = debug.setmetatable,
+				setlocal = debug.setlocal,
+				traceback = debug.traceback,
+				setfenv = debug.setfenv,
+				getinfo = debug.getinfo,
+				setupvalue = debug.setupvalue,
+				getlocal = debug.getlocal,
+				getregistry = debug.getregistry,
+				getfenv = debug.getfenv,
+			},
```

Comment:
If this is the entire `debug` table without modification, you could do just `debug = debug` instead of explicitly typing all of the members. See `math`, `string` and `table` below.